terraform/modules/hub/hub_policy.tf

```diff
@@ -6,7 +6,8 @@ module "policy_ecs_asg" {
   cluster             = "policy"
   vpc_id              = "${aws_vpc.hub.id}"
   instance_subnets    = ["${aws_subnet.internal.*.id}"]
-  number_of_instances = "${var.number_of_availability_zones}"
+  # number_of_instances = "${var.number_of_availability_zones}"
+  number_of_instances = 0
```

Comment:
How does `number_of_instances` relate to line 37, `number_of_tasks`? A task runs on an instance, so if there's 1 task but 0 instances then it won't work?

Reply:
in this case it just wont try and schedule a task as there is no place to schedule it